packages/origin-backend-client/src/DeviceClient.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+import { IDevice } from '@energyweb/origin-backend-core';
+
+export interface IDeviceClient {
+    getById(id: number): Promise<IDevice>;
+    getAll(): Promise<IDevice[]>;
+    add(id: number, device: IDevice): Promise<IDevice>;
+}
+
+export class DeviceClient implements IDeviceClient {
+    private baseURL: string;
+
+    constructor(_baseUrl: string) {
+        this.baseURL = _baseUrl;
+    }
+
+    private get endpoint() {
+        return `${this.baseURL}/Device`;
+    }
+
+    public async getById(id: number): Promise<IDevice> {
+        const url = `${this.endpoint}/${id}`;
+        const { data } = await axios.get(url);
```

Comment:
Please use `RequestClient` class from backend, instead of invoking axios directly. One of the reasons for using `RequestClient` is better way to configure request parameteres, like authentication token - for authorized XHR calls.